src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveRenderedService.cs

```diff
@@ -96,6 +97,9 @@ public async Task<RetrieveRenderedResponse> RetrieveRenderedImageAsync(RetrieveR
             sw.Stop();
             _logger.LogInformation("Render from dicom to {OutputContentType}, uncompressed file size was {UncompressedFrameSize}, output frame size is {OutputFrameSize} and took {ElapsedMilliseconds} ms", outputContentType, stream.Length, resultStream.Length, sw.ElapsedMilliseconds);
 
+            _dicomRequestContextAccessor.RequestContext.IsRenderedRequested = true;
```

Comment:
Not sure why needed this boolean, if the length is greater than 0, we can bill them. Whats your thought?

Reply:
yeah thats true, was just following the same logic as we have for transcode - but I dont see any real need for it in workspace-platform side either